### PR TITLE
Translations working now for Application forms

### LINKED ISSUE #932: Application link breaks when language is selected
When I select a language (Spanish, Chinese or Vietnamese)
And I click the Apply button, 
Then it takes me to a Page Not Found instead of the start of the application 
This is the URL: https://bloom.exygy.dev/vi/viapplications/start/choose-language?listingId=e5e94753-69a4-459a-bde2-55b9375fed40

### PR DESCRIPTION
**Pull request by jaredcwhite**
Fixes #932, #933

@kathyccheng We apparently didn't have a word meaning "Begin" for Vietnamese. I chose "Bắt đầu" from the online translator but we'll want that QA'd.

### CODE REVIEW

**Comment by kathyccheng** [comment]
@jaredcwhite I noticed a couple of things 

* When I first got to homepage in the deploy preview environment, clicked into a listing, clicked to apply online and then tried to choose to begin in a different language (Spanish, Vietnamese, or Chinese) on that first page of the application, when I clicked Begin, it went to the what to expect page and then when I clicked Next, it routed back to the homepage. I tried it over and over again in other languages and it wouldn't take me past the What to Expect. I kept get routed back to the homepage. But then, when I clicked Begin in English and then clicked Next on the What to Expect, it worked just fine. And then when I went back and tried the other languages, it then worked. 
* Also I noticed that in the Spanish path, each of the pages in Spanish continued in Spanish until I got to the household member screen and then it switched into English and continued in English from there 
* But looks like Apply Online seems to be working in each of the languages now!

I'll pass along the Vietnamese Begin to the translators.

File: sites/public/pages/applications/start/choose-language.tsx [hunk 1]
```diff
@@ -128,6 +128,17 @@ export default () => {
                   開始
                 </Button>
               )}
+
+              {listing?.applicationConfig.languages.some((lang) => lang == "zh") && (
```
Comment (dominikx96) [inline sites/public/pages/applications/start/choose-language.tsx:135]: Maybe will be better to save lang codes as enum and use it for application conductor and here, what do you think?

**Review by dominikx96 — COMMENTED** [review]
@jaredcwhite see my comment, but looks good! :)

**Comment by jaredcwhite** [comment]
I think all the latest issues with this PR have been fixed now. Please review and let me know! https://deploy-preview-952--clever-edison-cd22c1.netlify.app/

**Review by kathyccheng — COMMENTED** [review]
It looks good to me!!

**Pull request merged**